File: PHP/PROGRAM/mahasiswa.php
<?php

class mahasiswa{
// private atribut
private $nama = "";
private $NIM = "";
private $prodi = "";
private $fakultas= "";
private $gambar = "";
function __construct($nama="",$NIM="",$prodi="",$fakultas="", $gambar="") { // CONSTRUCTOR

$this->nama = $nama;
$this->NIM = $NIM;
$this->prodi = $prodi;
$this->fakultas = $fakultas;
$this->gambar = $gambar;
}
 # Getter and Setter
function setnama ($nama="") {
$this->nama = $nama;
 
}
function setNIM ($NIM="") {
    $this->NIM = $NIM;
}
function setprodi ($prodi="")  {
    $this->prodi= $prodi;
     
}
    function setfakultas($fakultas="") {
        $this->fakultas= $fakultas;
    }

    function setGambar($gambar = ""){
        $this->gambar = $gambar;
    }
    function getnama () {
    return $this->nama;
    }
    function getNIM() {
    
    return $this->NIM;
    }
    function getprodi () {
        return $this->prodi;
        }
    function getfakultas() {
        
        return $this->fakultas;
        }
    function getGambar(){
        return $this->gambar;
    }
}
?>

File: PY/PROGRAM/mahasiswa.py
class Mahasiswa:
    def __init__(self, Nama="", Nim="", prodi="", fakultas=""):  #
        self.Nama = Nama
        self.Nim = Nim
        self.prodi = prodi
        self.fakultas = fakultas

    # Getter and Setter
    def set_Nama(self, Nama):
        self.Nama = Nama

    def set_Nim(self, Nim):
        self.Nim = Nim

    def set_prodi(self, prodi):
        self.prodi = prodi

    def set_fakultas(self, fakultas):
        self.fakultas = fakultas

    def get_Nama(self):
        return self.Nama

    def get_Nim(self):
        return self.Nim

    def get_prodi(self):
        return self.prodi

    def get_fakultas(self):
        return self.fakultas
